src/main.rs

```diff
@@ -1,3 +1,829 @@
+            ),
+        }),
+        audits: None,
+    };
+
+    let workspace_config = || -> Option<Config> {
+        // FIXME: what is `store.path` relative to here?
+        Config::deserialize(metadata.workspace_metadata.get(WORKSPACE_VET_CONFIG)?)
+            .map_err(|e| {
+                error!(
+                    "Workspace had [{WORKSPACE_VET_CONFIG}] but it was malformed: {}",
+                    e
+                );
+                std::process::exit(-1);
+            })
+            .ok()
+    }();
+
+    let package_config = || -> Option<Config> {
+        // FIXME: what is `store.path` relative to here?
+        Config::deserialize(metadata.root_package()?.metadata.get(PACKAGE_VET_CONFIG)?)
+            .map_err(|e| {
+                error!(
+                    "Root package had [{PACKAGE_VET_CONFIG}] but it was malformed: {}",
+                    e
+                );
+                std::process::exit(-1);
+            })
+            .ok()
+    }();
+
+    let mut configs = vec![default_config];
+    if let Some(cfg) = workspace_config {
+        configs.push(cfg);
+    }
+    if let Some(cfg) = package_config {
+        configs.push(cfg);
+    }
+    let config = Configs(configs);
+
+    info!("Final Config: ");
+    info!("  - version: {}", config.version());
+    info!("  - store.path: {:#?}", config.store_path());
+    info!("  - audits: {:#?}", config.audits());
+
+    //////////////////////////////////////////////////////
+    // Run the actual command
+    //////////////////////////////////////////////////////
+
+    let init = is_init(&config);
+    if matches!(cli.command, Some(Commands::Init { .. })) {
+        if init {
+            error!(
+                "'cargo vet' already initialized (store found at {:#?})",
+                config.store_path()
+            );
+            std::process::exit(-1);
+        }
+    } else if !init {
+        error!(
+            "You must run 'cargo vet init' (store not found at {:#?})",
+            config.store_path()
+        );
+        std::process::exit(-1);
+    }
+
+    match &cli.command {
+        Some(Commands::Init(sub_args)) => cmd_init(out, &cli, &config, &metadata, sub_args),
+        Some(Commands::Fetch(sub_args)) => cmd_fetch(out, &cli, &config, &metadata, sub_args),
+        Some(Commands::Certify(sub_args)) => cmd_certify(out, &cli, &config, &metadata, sub_args),
+        Some(Commands::Forbid(sub_args)) => cmd_forbid(out, &cli, &config, &metadata, sub_args),
+        Some(Commands::Suggest(sub_args)) => cmd_suggest(out, &cli, &config, &metadata, sub_args),
+        Some(Commands::Diff(sub_args)) => cmd_diff(out, &cli, &config, &metadata, sub_args),
+        Some(Commands::Audits(sub_args)) => cmd_audits(out, &cli, &config, &metadata, sub_args),
+        Some(Commands::HelpMarkdown(sub_args)) => cmd_help_markdown(out, &cli, &config, &metadata, sub_args),
+        None => cmd_vet(out, &cli, &config, &metadata),
```

Comment:
Any subcommands I missed?

Reply:
I recently added `accept-criteria-change`, and per above I don't think `audits` is a thing.